### DISCUSSION TITLE
Orario Docente

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#534] [Idee e proposte]
Sarebbe possibile sviluppare una funzione che mostri l'orario del docente per andare a firmare dove è presente in orario?

**Comment by trinko** [comment]
La segno come richiesta.

**Reply by manuelbarberi** [reply]
ma verrà aggiunta per il prossimo anno vero? Perchè nella scuola comunque sarebbe una cosa utilissima.

**Comment by trinko** [comment]
Sono d'accordo con te che sia utile, perché spesso i docenti dimenticano qualche firma o sbagliano firmando l'ora di un collega.
Ma non sono proprio in grado di prevedere quando potrà essere sviluppata.
